### DISCUSSION TITLE
How Can I make the Dark Mode Activated for All Users?

### DISCUSSION THREAD

**Discussion by Alkebsi** [community/community#38506] [Other Feature Feedback, Questions, & Ideas] [answered]
I have been customizing my profile lately and saw that the dark mode only applies to how I see my profile, not how others see it. I have added a lot of white SVGs that have never appeared to the others, since they had their Github theme set as "Light". Wish I could have an option to make it as I see it to all users.

**Comment by subidit** [comment, marked as answer]
You could try using svg or png with transparent background. That way it won't matter if the viewer is using light or dark theme. 
Hope it helps. 🙏

**Reply by david-engelmann** [reply]
bump, you can't modify other peoples appearance settings

**Reply by Alkebsi** [reply]
> bump, you can't modify other peoples appearance settings

I know that at the moment! I am requesting for a change if possible or, as I would do, follow @subidit suggestion.

**Reply by david-engelmann** [reply]
I'd suggest following @subidit suggestion. Changing peoples themes is going to violate digital accessibility standards

**Comment by fabioschorn** [comment]
It's not possible yet... A shame, because my profile is configured with parts in darkmode...

**Reply by neil-edelman** [reply]
The work-around is of limited use with graphs with transparency. Dark mode and black graph elements are just not visible, similarly the other way around. When it is important, one should be able to suggest a mode, only overwriting if the user has explicitly set one or the other for accessibility.
